test/integration/smoke/test_primary_storage.py

```diff
@@ -479,6 +478,8 @@ def setUpClass(cls):
                 hypervisor=cls.hypervisor,
                 mode=cls.zone.networktype
             )
+            cls._cleanup.append(cls.virtual_machine_1)
+            cls._cleanup.append(cls.service_offering_1)
```

Comment:
again, should not happen here, look at the tearDownClass implementation

Reply:
maybe
```suggestion
            cls._cleanup.append(cls.virtual_machine_1)
```
 was meant?